src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,9 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static Color getRandomColor() {
+        return Color.values()[new Random().nextInt(6)];
```

Comment:
Think about which variables should be local in the method and which should be class-level
Let’s  make 6 - the constant variable 
new Random() - class level variable